README.ci.md

```diff
@@ -1,14 +1,14 @@
 **WARNING:** This document is a work in progress and intended as a RFC.
-If you are not a Coq Developer, don't follow this instructions yet.
+If you are not a Coq Developer, don't follow these instructions yet.
 
 Introduction
 ============
 
-The Coq Travis CI infrastructure is meant to provide lightweight
-automatics testing of pull requests.
-If you are on GitLab, their integrated CI is also set up.
+As of 2017, lightweight tests are automatically run against all pull requests
+or merge requests submitted to Coq's official GitHub or GitLab repositories,
```

Comment:
@Zimmi48, thanks for explaining and sorry I misunderstood :) Is the latest commit any better?